src/java/htsjdk/samtools/liftover/LiftOver.java

```diff
@@ -156,8 +156,8 @@ private static Interval createToInterval(final String intervalName, final Target
             toEnd = negativeEnd;
         }
         // Convert to 1-based, inclusive.
-        return new Interval(targetIntersection.chain.toSequenceName, toStart+1, toEnd, targetIntersection.chain.toNegativeStrand,
-                intervalName);
+        final boolean strand = targetIntersection.chain.toOppositeStrand ? !sourceNegativeStrand : sourceNegativeStrand;
```

Comment:
Hi Tim,

(Please ignore smell of burning dust from this portion of my memory ;-)

One tiny niggle: could you change the name of this variable from 'strand' to 'negativeStrand?'

Other than that, :+1: 

-Alec